frame/bridge/relayer-game/src/lib.rs

```diff
@@ -161,6 +166,15 @@ decl_storage! {
 			get(fn bonds_of_relayer)
 			: map hasher(blake2_128_concat) AccountId<T>
 			=> RingBalance<T, I>;
+
+
+		// TODO: move into relay
```

Comment:
We can have a discussion before develop guard system, we can leave it in the next PR.